scraper/transfers/file_utils.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import csv
+
+from .data_types import Class_Object
+from ..Logger import Logger
+
+
+class File_Utils(object):
+
+    """
+    File utilities for writing and reading data from files
+    """
+
+    def __init__(self):
+        """
+        Blank init
+        """
+        self.log = Logger(self.__class__.__name__)
+
+    def write_to_csv(self, class_obj_list, output_file_name):
+        """
+        Writes the class_obj_list to a a value quoted CSV file
+
+        :class_obj_list: Class Object list to get data from
+        :output_file_name: Name of the file to write to
+
+        :returns: None
+        """
+        with open(output_file_name, 'w') as csv_file:
```

Comment:
Like @Blu3spirits said earlier, if you use built-in models you may not have to re-define the field names here.

Reply:
This was done *long* before the Django model framework even existed.  It'll be something that can be looked at later.